src/ListView/ListViewRow.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import {
+  ListViewActions,
+  ListViewAdditionalInfo,
+  ListViewCheckbox,
+  ListViewLeft,
+  ListViewBody,
+  ListViewDescription,
+  ListViewDescriptionHeading,
+  ListViewDescriptionText,
+  ListViewMainInfo
+} from './index'
+
+/**
+ * ListViewRow wraps the ListViewItem row, conditionally renders sections
+ * based on availability of individual props, maintains the ListItem row structure
+ */
+const ListViewRow = ({
+  actions,
+  additionalItems,
+  checkboxInput,
+  leftContent,
+  heading,
+  description
+}) =>
+  <div style={{ display: 'flex', flex: 1 }}>
```

Comment:
I’m curious why you included the inline styles here. The only difference I see in layout when these styles are applied vs removed is that the border between the checkbox and title is either the height of the `list-view-pf-main-info` or the height of the `list-view-pf-checkbox`. This is the only visual difference I can see with the core implementation. 

Did you have an issue with the layout when these styles were not included?

Reply:
Omitting the styling breaks the flexbox chain and causes the row not to span across whole width of parent. This effect is visible in storybook ListView > ListView variants - 3rd item - if you remove the styling there, it will shorten to the left, making the title very narrow.

This styling is needed because ListViewRow adds additional div. I know that bootstrap 4 has a special class for this, bootstrap 3 does not.

Reply:
Thanks for the clarification! I see what you're referring to now. My only suggestion would be to include these styles under a class, that way we can use this class in other components where similar issues occur. I'm fine with that being tracked as a separate issue outside this PR, though.